conda_build/build.py

```diff
@@ -432,6 +432,10 @@ def write_info_json(m):
             info_index['depends'] = [' '.join(dist.split('::', 1)[-1].rsplit('-', 2))
                                      for dist in dists]
 
+    # change index.json timestamp for reproducibility
+    if 'timestamp' in info_index:
+        info_index['timestamp'] = 86400
```

Comment:
Great point. I also set the default to 0 instead of 86400 as the next PR will contain a patch that sets pyc file timestamps to 0.